userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -933,31 +933,262 @@ To delete a payment method:
+
+| Immediate Creation
+| Select this option to start the subscription immediately.
+
+| Specify a date
+| To define a specific date on which the subscription begins, select this option, and choose a date from the *Date* field that appears.
+
+|===
+
+==== Pause/Resume Subscriptions
+
+Pausing and resuming happens at the subscription bundle level. In other words, pause/resume affects _all_ subscriptions and add-ons in the customer's account.
+
+[NOTE]
+*Note:* While a customer's subscription bundle is paused, you cannot add any new subscriptions or add-ons to the account.
+
+You can use the pause/resume feature to:
+
+* Stop a subscription bundle indefinitely
+* Reactivate a paused subscription bundle
+* Set a specific time period during which the bundle is paused and then automatically resumed
+
+1. Open the account and click *Subscriptions* on the sub-menu.
+2. Hover over the gift icon and click *Pause / Resume*:
+
+image::Subscriptions-SelectPauseResume.png[width=350,align="center"]
+
+Kaui opens the Pause/Resume screen:
+
+image::Subscriptions-PauseResume.png[width=550,align="center"]
 
-// This area of Kaui lists the subscriptions associated with the account.
+[start=3]
+. Fill in the following fields:
+
+* *Pause date*—The calendar date on which to temporarily stop the subscription bundle. To resume a paused subscription, leave this field blank.
+* *Resume date*—The calendar date on which to remove the pause from the subscription(s). To set a pause to continue indefinitely, leave this field blank.
+* *Comment*—The text you enter here displays on the <<Timeline>> page after saving the pause/resume. Adding comments is optional.
+
+[start=4]
+. Click the *Pause / Resume* button. Kaui displays a success message at the top of the screen.
+
+==== Change a Subscription
+
+In Kaui, you can change a customer's subscription from one plan to another plan. You can also define a different price than what is set in the catalog and when the change takes effect.
+
+To change an account's subscription:
+
+1. Open the account and click *Subscriptions* on the sub-menu.
+2. In the *Actions* column (far right), click *Change*.
+
+image::SelectChangeSubscription.png[width=250,align="center"]
+
+Kaui opens the Change Subscription screen:
+
+image::ChangeSubscription.png[width=450,align="center"]
+
+[start=3]
+. In the *New plan* field, select a different plan.
+. If you need to override the price for the plan you've selected, enter the amount in the *Price Override* field.
+. To accept the system's default policy for changing subscriptions (`END_OF_TERM`), leave *Default policy* selected and click *Save.*
+
+OR
+
+You can select one of two other options: *Specify Policy* or *Specify a date*. This will override the policy defined in the catalog for the plan. (For information on this topic, see see https://docs.killbill.io/latest/userguide_subscription.html#_plan_change_timing["Plan Change Timing"] in the _Subscription Guide_.)
+

```

Comment:
The URL is not rendered properly in the asciidoc file. See https://github.com/FlowWriter/killbill-docs/blob/kaui-only-5/userguide/kaui/Kaui_Guide_Draft.adoc#change-a-subscription